pages/participate/participate.md

```diff
@@ -46,6 +46,7 @@ Accepted abstracts will be placed on the website ahead of the conference.
 - **Papers and Notebooks**: Due Monday, June 3. Notifications Wednesday, July 10 for notebooks and Wednesday, July 17 for papers.
 - **Posters and Talks**: Due Monday, June 17 AoE. Notifications Friday, July 19.
 - **Camera-ready submissions (Notebooks, Papers)**: Friday, August 23
+- **Camera-ready submissions (Posters)**: Friday, October 25
```

Comment:
Please also add this to: https://github.com/USRSE/usrse24/blob/main/pages/about/dates.md

Reply:
Done